.pre-commit-config.yaml

```diff
@@ -33,6 +31,8 @@ repos:
           - '--args=--only=terraform_required_providers'
           - '--args=--only=terraform_standard_module_structure'
           - '--args=--only=terraform_workspace_remote'
+      - id: terraform_validate
```

Comment:
just moved lower in the checks since it takes the longest - the shortest checks will return first before the longer checks